google-cloud-video_intelligence-v1p3beta1/.owlbot-manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "generated": [
+    ".gitignore",
+    ".repo-metadata.json",
+    ".rubocop.yml",
+    ".yardopts",
+    ".yardopts-cloudrad",
+    "AUTHENTICATION.md",
+    "CHANGELOG.md",
+    "Gemfile",
+    "LICENSE.md",
+    "README.md",
+    "Rakefile",
+    "gapic_metadata.json",
+    "google-cloud-video_intelligence-v1p3beta1.gemspec",
+    "lib/google-cloud-video_intelligence-v1p3beta1.rb",
+    "lib/google/cloud/video_intelligence/v1p3beta1/version.rb",
+    "lib/google/cloud/videointelligence/v1p3beta1.rb",
+    "lib/google/cloud/videointelligence/v1p3beta1/streaming_video_intelligence_service.rb",
+    "lib/google/cloud/videointelligence/v1p3beta1/streaming_video_intelligence_service/client.rb",
+    "lib/google/cloud/videointelligence/v1p3beta1/streaming_video_intelligence_service/credentials.rb",
+    "lib/google/cloud/videointelligence/v1p3beta1/video_intelligence_pb.rb",
+    "lib/google/cloud/videointelligence/v1p3beta1/video_intelligence_service.rb",
+    "lib/google/cloud/videointelligence/v1p3beta1/video_intelligence_service/client.rb",
```

Comment:
In the previous versions of this client, the module name was `google/cloud/video_intelligence` instead of `google/cloud/videointelligence`. Is this significant naming change intentional?

```
google-cloud-video_intelligence-v1p2beta1/lib/google/cloud/video_intelligence/v1p2beta1/video_intelligence_service/client.rb
```

Reply:
Oh, shoot, thanks for catching that. We're missing a proto option in this version.